app/packages/core/src/plugins/SchemaIO/components/SliderView.tsx

```diff
@@ -245,20 +241,20 @@ export default function SliderView(props) {
             <Grid container justifyContent="space-between" pl={1}>
               <Grid item pl={labelPosition === "left" ? "100px" : "0"}>
                 <SliderInputField
-                  label="Min"
+                  label="Min %"
```

Comment:
Instead of harcoding this, can we just make this a variable? So if someone in Python defines valueFormat = %, we can map this in as label[valueFormat]

Idk if we’ll end up adding icons as labels instead like talked about in Slack, or even just text like “units”, but this would make it more flexible

Reply:
makes sense - I included the unit at minimum for now - we should ask design how she wants to handle this